apps/src/code-studio/pd/foorm/editor/form/FoormFormSaveBar.jsx

```diff
@@ -294,11 +299,22 @@ class FoormFormSaveBar extends Component {
               Publish
             </button>
           )}
+          {this.props.isFormPublished && (
```

Comment:
Nice, so we show this only once a Form has been published? Good call

Reply:
I don't know why I thought of this, but I realized we probably need to let people bump the version on unpublished forms. 

I think the flow you'd want is take the existing Form, bump the version, and edit it in an unpublished (ie, draft) state before it goes live. Then, you publish when you want it to take over the existing version.

If we take the example that's happening now (post-workshop survey), if we require publishing before versioning, a survey editor wouldn't be able to work on the new version without that new version being automatically put in place of the previous version. @tess323 FYI, does this make sense?

Reply:
Oh yeah, that definitely makes sense. Ok so: only on the latest version of a form we show the "save as new version" button, published or not

Reply:
That makes sense to me!

Reply:
mmm I would check though that we have logic to grab the most recent **published** version when rendering a, say, post-workshop survey, at a glance it looks like we may not do that?

https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/foorm/form.rb#L93

Reply:
thanks for the heads up - thank does make sense to me